core-java-modules/core-java-serialization/src/test/java/com/baeldung/deserialization/DeserializationUnitTest.java

```diff
@@ -24,7 +25,7 @@
     @Test
     public void testDeserializeObj_compatible() throws IOException, ClassNotFoundException {
 
-        assertEquals(userDefinedSerialVersionUID, AppleProduct.getSerialVersionUID());
+        Assert.assertEquals(userDefinedSerialVersionUID, AppleProduct.getSerialVersionUID());
```

Comment:
assertEquals and assertNotEquals are already statically imported - let's use them this way.